modules/ROOT/pages/tuning-guide.adoc

```diff
@@ -0,0 +1,25 @@
+= Performance Tuning Guide
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Running Mule applications in their optimally performing state requires a careful review of not only the applications but also the environment where they run. Mule 4 was designed to tune itself for optimal performance depending on several factors; however, this does not prevent your applications from experiencing performance issues because of their construction or dependencies. Similarly, for on-premises installations, the environment itself can be tuned so the Mule application can take full advantage of it. Many variables base the performance of your application, and tuning these variables still requires a trial-and-error process. +
+To make this process easier, the tuning guide highlights performance best practices and the ideal test environments to run applications. These recommendations are collected from the Services Engineering team,  in-house benchmarking efforts by the MuleSoft Performance Engineering, and from the Development team.
```

Comment:
```suggestion
You can simplify performance tuning by using documented best practices and testing your applications in customized environments. The following recommendations come from the Development and Services Engineering teams and benchmarking efforts by MuleSoft Performance Engineering.
```
I reordered the teams based on the order of their provided services: dev first, then service eng., then performance eng?